vivo-project/VIVO
# Vivo 1287

Update to PR - https://github.com/vivo-project/VIVO/pull/108
Only change was to incorporate Brian Lowes recommendation to remove the subclass declarations and let 4 of the 5 affected objects just roll up to owl:Thing. 
dateTimePrecision incorporated Mikes subClass declaration of quality. This shouldn't affect anything in the application.
Tested this on a VM.
Original PR Notes below
----------------------------------------
VIVO-1287

What does this pull request do?
Provides new parent classes for five classes. These five classes were classed as skos:Concept causing VIVO to return instances in the list of Concepts, an undesirable behavior, because the classes are not skos:Concepts

What's new?
AcademicDegree - not a subclass
Award - not a subclass
Credential - not a subclass
ARG_2000376 - not a subclass
DateTimePrecision - subClass of quality
How should this be tested?
Build (no need to clean out data)
Review existing instances of degrees, awards, credentials, and dates -- each should display and edit normally. ARG_2000376 is not used by the VIVO application, no testing is needed.
Create new degrees, awards, credentials and dates. Each should create and display normally.
View the Index to see that instances of these classes no longer appear in the index under Concept.
Additional Notes:
The Class diagram in the VIVO Technical documentation will need to be updated when software is released. The ontology diagrams should not need to be updated -- there is no change in the models

Sites with existing instances of these classes (that's all existing VIVO sites) may wish to implement CONSTRUCTs to update the parent classes for existing instances (these can be provided so that sites do not have to each write them) when this software is released. New instances will not appear in the concept list. Old instances will continue to appear until CONSTRUCT is run.
Interested parties
@vivo-project/vivo-committers @vivo-project/contributors @vivo-project/vivo-ontologists
